client/components/Links.spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import {
+  renderIntoDocument,
+  findRenderedDOMComponentWithClass,
+  scryRenderedDOMComponentsWithClass,
+  scryRenderedComponentsWithType
+} from 'react-addons-test-utils';
+import { expect } from 'chai';
+
+import { Links } from './Links';
+
+describe('Links Component', () => {
+  let component;
+  let git;
+  let stack;
+
+  beforeEach('render the component', () => {
+    git = ['i\'m just a link', 'yes i\'m only a link'];
+    stack = ['and i\'m sitting here on capitol hill'];
+
+    component = ReactTestUtils.renderIntoDocument(<Links
```

Comment:
Want to fool around with this, figure out why it would fail as a functional component.